src/sql/src/vars.rs

```diff
@@ -17,16 +17,46 @@ use once_cell::sync::Lazy;
 use serde::Serialize;
 use uncased::UncasedStr;
 
+use crate::ast::Ident;
+use crate::DEFAULT_SCHEMA;
 use mz_build_info::BuildInfo;
 use mz_ore::cast;
 use mz_persist_client::cfg::PersistConfig;
-use mz_sql::ast::Ident;
-use mz_sql::DEFAULT_SCHEMA;
 use mz_sql_parser::ast::TransactionIsolationLevel;
 
-use crate::catalog::SYSTEM_USER;
-use crate::error::AdapterError;
-use crate::session::{EndTransactionAction, User};
+/// The action to take during end_transaction.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum EndTransactionAction {
+    /// Commit the transaction.
+    Commit,
+    /// Rollback the transaction.
+    Rollback,
+}
```

Comment:
I don't think it makes a ton of sense for this enum to live in this file, but I'm not sure if there's a better place. 

It might be OK to duplicate this enum here and back in it's original location, and make a `From` impl in the `session.rs` to help convert between the two.

Reply:
Chatted over slack, leaving here for now with a comment b/c we don't know a better way